src/qtcore/qml/UpdateGeometry.js

```diff
@@ -194,6 +196,38 @@ function updateVGeometry(newVal, oldVal, propName) {
         this.height = height;
 
     this.$updatingGeometry = false;
+
+    if (this.parent != undefined) updateChildrenRect(this.parent);
 }
 
+function updateChildrenRect(component) {
+    var children = component !== undefined ? component.children : undefined
+    if ( children == undefined || children.length == 0 )
+        return;
+
+    var maxWidth = 0;
+    var maxHeight = 0;
+    var minX = children.length>0 ? children[0].x : 0;
+    var minY = children.length>0 ? children[0].y : 0;
+    var h=0;
+    var w=0;
+    var child;
+
+    for (var i=0;i<children.length; i++){
+        child = children[i];
+
+        h = child.$isUsingImplicitHeight ? child.implicitHeight : child.height;
+        w = child.$isUsingImplicitWidth ? child.implicitWidth : child.width;
```

Comment:
@akreuzkamp, it is also used in a many more places at c1bf9083d7beabf53d676ec2f187f82cde973e97.

Are all those `child.$isUsingImplicitHeight ? child.implicitHeight : child.height` and `child.$isUsingImplicitWidth ? child.implicitWidth : child.width` unnecessary?

/cc @igosoft